### DISCUSSION TITLE
Coreshop default email with multiple language

### DISCUSSION THREAD

**Discussion by rushitkhunt1996** [coreshop/CoreShop#2320] [Q&A] [answered]
Hello, @dpfaffenbauer , @everyone
We use the Coreshop for eCommerce website, and now we face an issue with email, the email notification is working fine for registration, reset password, and all with English, but when we change the language at that time these email notifications are not working.
we are using country base language so we have document structure is like,
![image](https://github.com/coreshop/CoreShop/assets/60934941/8b20fcd7-25c3-4861-abc1-fd4fcbcfc153)

is there any configuration? please suggest.

**Comment by dpfaffenbauer** [comment, marked as answer]
Did you configure all the language emails in the notification rules?

**Reply by rushitkhunt1996** [reply]
no, i missed that, let me do this, I think it should work.

**Reply by rushitkhunt1996** [reply]
Yes, @dpfaffenbauer  it working fine, Thanks for your support.
